src/form-generator/i18n/form-functions.en.html

```diff
@@ -0,0 +1,291 @@
+        <div><a href="/feed/?f=1" target="_blank">Forumotion Update Feed</a></div>
+        <div><a href="https://www.forumotion.com/benefits-of-forums" target="_blank">Benefits of having a forum</a></div>
+        <div><a href="https://www.forumotion.com/create-forum/phpbb3" target="_blank">Create a free forum</a></div>
+      </div>
+
+      <div class="bloc">
+        <strong class="h3">Services</strong>
+        <div><a href="/h7-" target="_blank">Forum Support</a></div>
+        <div><a href="/f6-" target="_blank">Graphic Section</a></div>
+        <div><a href="/f60-" target="_blank">Review Section</a></div>
+        <div><a href="/t5855-" target="_blank">Forum Promotion</a></div>
+        <div><a href="http://en.hitskin.com/" target="_blank">Free Forum Themes</a></div>
+      </div>
+
+      <div class="bloc">
+        <strong class="h3">Follow us</strong>
+        <div><a href="https://www.facebook.com/forumotionpage" target="_blank"><i class="fa fa-facebook"></i> Facebook</a></div>
+        <div><a href="https://twitter.com/Forumotion" target="_blank"><i class="fa fa-twitter"></i> Twitter</a></div>
+        <div><a href="https://plus.google.com/+forumotion" target="_blank"><i class="fa fa-google-plus"></i> Google+</a></div>
+        <div><a href="https://www.pinterest.com/forumotion/" target="_blank"><i class="fa fa-pinterest"></i> Pinterest</a></div>
+        <div><a href="https://www.youtube.com/user/ForumotionDotCom" target="_blank"><i class="fa fa-youtube-play"></i> Youtube</a></div>
+      </div>
+
+      <div class="bloc">
+        <strong class="h3">Information</strong>
+        <div><a href="/t368-" target="_blank">Forumotion Support Team</a></div>
+        <div><a href="/t131169-" target="_blank">Design Section Rules</a></div>
+        <div><a href="/t59451-" target="_blank">Frequently Asked Questions</a></div>
+        <div><a href="/t61210-" target="_blank">Tips and Tricks</a></div>
+        <div>Generator made by <u><a href="https://ajuda.forumeiros.com/u60563" target="_blank">Luiz Felipe</a></u>.
+      </div>
+
+      <div style="clear:both"></div>
+      <hr />
+      <div class="footer-link">
+        <div><a href="https://www.forumotion.com/create-forum/phpbb3" target="_blank"><span>Create a free forum</span></a></div>
+        <div><img src="https://i.servimg.com/u/f38/18/21/41/30/free-h10.png" alt="Free forum" /></div>
+        <div><a href="https://www.forumotion.com/" target="_blank"><span>© 2020, forumotion.com</span></a></div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Javascript's -->
+  <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js"></script>
+  <script type="text/javascript" src="https://forumeiros.github.io/assets/src/form-generator/form-functions.js"></script>
```

Comment:
Já que este é o HTML do formulário de inglês, aponte para o código `i18n/form-functions.en.js`.

```suggestion
  <script type="text/javascript" src="https://forumeiros.github.io/assets/src/form-generator/i18n/form-functions.en.js"></script>
```